keras_cv/models/backbones/densenet/densenet_backbone_test.py

```diff
@@ -92,24 +92,28 @@ def test_saved_alias_model(self, save_format, filename):
         restored_output = restored_model(self.input_batch)
         self.assertAllClose(model_output, restored_output)
 
-    def test_create_backbone_model_from_alias_model(self):
-        model = DenseNet121Backbone(
-            include_rescaling=False,
-        )
+    def test_feature_pyramid_inputs(self):
+        model = DenseNet121Backbone()
         backbone_model = get_feature_extractor(
             model,
             model.pyramid_level_inputs.values(),
             model.pyramid_level_inputs.keys(),
         )
-        inputs = tf.keras.Input(shape=[256, 256, 3])
+        input_size = 256
+        inputs = tf.keras.Input(shape=[input_size, input_size, 3])
         outputs = backbone_model(inputs)
-
-        levels = ["P2", "P3", "P4"]
-        self.assertLen(outputs, 3)
-        self.assertEquals(list(outputs.keys()), levels)
-        self.assertEquals(outputs["P2"].shape, [None, 32, 32, 128])
```

Comment:
good catch -- one question though:

wdyt about keeping these assertions as-is but using `input_size / 2 ** {2/3/4}` as the width/height fields?

I personally find the `for` loop approach a bit harder to parse, so it makes the test assertion a little less explicit to me.

Reply:
How would you handle different models supporting different P-levels? I guess you would just use the expected ones?

Reply:
Right -- I think the way we had it before made this pretty overt (we just have an assertion for every separate P-level)